OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -66,12 +66,20 @@ public class CarryallInfo : TraitInfo, Requires<BodyOrientationInfo>, Requires<A
 		[Desc("Condition to grant to the Carryall while it is carrying something.")]
 		public readonly string CarryCondition = null;
 
+		[ActorReference(dictionaryReference: LintDictionaryReference.Keys)]
+		[Desc("Conditions to grant when a specified actor is being carried.",
+			"A dictionary of [actor id]: [condition].")]
```

Comment:
Small nit: 
```suggestion
			"A dictionary of [actor_name]: [condition].")]
```

Reply:
This desc is mostly copied from Cargo. It says actor id there too, should i update both (and any other instances if we have more)? I would rather keep things consistent.

Reply:
Well I was confused by "actor ID" because it's totally not what I was expecting, so if you could and it's reasonable to sneak into this PR then yes, please. Otherwise I guess I'll have to live with this.

Reply:
If you change this please use "actor name" without the underscore.